src/Components/Server/src/BlazorPack/BlazorPackHubProtocol.cs

```diff
@@ -78,7 +78,7 @@ public bool TryParseMessage(ref ReadOnlySequence<byte> input, IInvocationBinder
                     message = PingMessage.Instance;
                     return true;
                 case HubProtocolConstants.CloseMessageType:
-                    message = CreateCloseMessage(ref reader);
+                    message = CreateCloseMessage(ref reader, itemCount);
```

Comment:
@rynowak @pranavkm @mkArtakMSFT FYI I just merged this PR into release/3.1 which changes BlazorPackHubProtocol.

Reply:
Oof, probably should've gotten sign off from @pranavkm before the merge. Next time :). Can you take a quick look @pranavkm and see if this seems problematic?